server/models/Request.js

```diff
@@ -0,0 +1,40 @@
+const mongoose = require("mongoose");
+const bcrypt = require("bcryptjs");
+
+const requestSchema = new mongoose.Schema({
+  user_id: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "User",
+    required: true,
+  },
+  sitter_id: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "User",
+    required: true,
+  },
+  start_date: {
+    type: Date,
+    required: true,
+  },
+  end_date: {
+    type: Date,
+    required: true,
+  },
+  accepted: {
+    type: Boolean,
+    default: false,
+    required: true,
+  },
+  declined: {
+    type: Boolean,
+    default: false,
+    required: true,
+  },
```

Comment:
I think we can be more flexible and future-proof if we reduce these two into a single field for these two (e.g. `status`).

Reply:
fixed